lib/sentry.client.js

```diff
@@ -1,5 +1,6 @@
 import VueLib from 'vue'
 import * as Sentry from '@sentry/browser'
+import { Integrations as ApmIntegrations } from "@sentry/apm"
```

Comment:
I think we don't want to include this dependency unconditionally which means we have to use dynamic import and import it only when options say so.

Reply:
You are right.